ember/tests/unit/adapters/adapter-test.js

```diff
@@ -0,0 +1,12 @@
+import { setupTest } from "ember-qunit";
+import { module, test } from "qunit";
+
+module("Unit | Adapter | identity", function (hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test("it exists", function (assert) {
+    const adapter = this.owner.lookup("adapter:identity");
+    assert.ok(adapter);
+  });
```

Comment:
Maybe test the adapter since this should be very easy tests and the adapter is quite an important piece of code for the profile.

Reply:
Added tests

Reply:
Awesome thanks!